oot.sh

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/bash
+
+while getopts ':c:' OPT; do
+        case ${OPT} in
+        c)
+		case ${OPTARG} in
+		clang|gcc|linaro_gcc)
+			export _compiler=${OPTARG}
+			;;
+		*)
+			echo "Compiler '${OPTARG}' unknown or not implemented"
+			exit
+			;;
+		esac
+		;;
+        ?)
+                echo && echo "Incorrect option: ${OPT}"
+                exit
+                ;;
+        esac
+done
+
+shift $(( OPTIND - 1 ))
+
+
+if (( "$#" == 0 )); then
+	echo "Usage: $0 <device> (Example for akatsuki: $0 akatsuki)"
+	exit
+fi
+
+
+# Default to clang
+if [[ -z "${_compiler}" ]]; then
+	export _compiler=clang
+fi
+
+
+# Platform common
+export BOARD_KERNEL_BASE=0x00000000
+export BOARD_KERNEL_PAGESIZE=4096
+export BOARD_KERNEL_TAGS_OFFSET=0x01E00000
+export BOARD_RAMDISK_OFFSET=0x02000000
+
+for _device in "$@"; do
+	unset BOARD_KERNEL_CMDLINE _platform _has_dtbo _recovery_ramdisk
+
+	BOARD_KERNEL_CMDLINE+=" lpm_levels.sleep_disabled=1"
+	BOARD_KERNEL_CMDLINE+=" service_locator.enable=1"
+
+	# Device specific
+	case ${_device} in
+	mermaid)
+		export _platform=ganges
+		;;
+	discovery|pioneer|voyager)
+		export _platform=nile
+		;;
+	bahamut|griffin)
+		export _platform=kumano
+		;;
+	pdx201)
+		export _platform=seine
+		;;
+	akari|akatsuki|apollo)
+		export _platform=tama
+		;;
+	lilac|maple|poplar)
+		export _platform=yoshino
+		;;
+	*)
+		echo "Device '${_device}' unknown or not implemented"
+		exit
```

Comment:
How about
```suggestion
		exit 1
```
here and in other places, to make chaining and in general communicating a failure state easier?

Reply:
Thanks for the advice :) Implemented.